src/index.tsx

```diff
@@ -233,9 +233,7 @@ const Routes: React.FC = () => {
                 component={NavigationSection}
               />
               <SectionRoute
-                permissions={[
-                  PermissionEnum.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES
-                ]}
+                permissions={[PermissionEnum.MANAGE_PRODUCTS]}
```

Comment:
And here similarly? `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES` sounds to be better for this section.